index.html

```diff
@@ -1475,6 +1475,28 @@ <h2>Relationship to Linked Data</h2>
 
       </section>
 
+      <section>
+        <h2>Relationship to Verifiable Credentials</h2>
+        <p>
+Cryptographic suites that implement this specification can be used to secure
+<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-credential">
+verifiable credentials</a> and
+<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-presentation">
+verifiable presentations</a>. Implementers
+that are addressing those use cases are cautioned that additional checks might
+be appropriate when processing those types of documents. For example, there are
+some use cases where it is important to ensure that the
+<a href="#proofs">`proof.verificationMethod`</a> value is associated with the
+<a data-cite="VC-DATA-MODEL-2.0#issuer">`issuer`</a> URL value in a
+<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-credential">
+verifiable credential</a>, or the
+<a data-cite="VC-DATA-MODEL-2.0#issuer">`holder`</a> URL value in a
+<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-credential">
+verifiable presentation</a>, during the process of
+<a data-cite="VC-DATA-MODEL-2.0#issuer-0">validation</a>.
```

Comment:
Hmm, some links here might not be quite right and we should be a bit more clear about the association.

```suggestion
<a>verification method</a> used in a proof is associated with the
<a data-cite="VC-DATA-MODEL-2.0#issuer">`issuer`</a> in a
<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-credential">
verifiable credential</a>, or the
<a data-cite="VC-DATA-MODEL-2.0#dfn-holders">`holder`</a> in a
<a data-cite="VC-DATA-MODEL-2.0#dfn-verifiable-presentation">
verifiable presentation</a>, during the process of
<a data-cite="VC-DATA-MODEL-2.0#issuer-0">validation</a>. One
way to check for such an association is to ensure that the value of the
<code>controller</code> property of a proof's <a>verification method</a>
matches the URL value used to identify the
<a data-cite="VC-DATA-MODEL-2.0#issuer">`issuer`</a> or
<a data-cite="VC-DATA-MODEL-2.0#dfn-holder">`holder`</a>, respectively.
This particular association indicates that the
<a data-cite="VC-DATA-MODEL-2.0#issuer">`issuer`</a> or
<a data-cite="VC-DATA-MODEL-2.0#dfn-holder">`holder`</a>, respectively,
are the controller of the <a>verification method</a> used to verify
the proof.
```